lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -326,6 +359,84 @@ static bool sinkCode(Function *F) {
       }
     }
 
+    // Sink Transpose below ChannelShuffle (Reshape->Transpose->Reshape)
+    if (auto *RN1 = dyn_cast<ReshapeNode>(node)) {
```

Comment:
nit: Can you use different names for the `RN1`/`RN2`/`TR1`/`TR2` here? It's a little hard to follow, especially because in the actual dataflow graph `RN2` comes before `RN1`, etc. given the current names.

E.g. something like `preShuffleRN`, `postShuffleRN`, `shuffleTR`, and `TR`.